tests/audit-tests/null/result.json

```diff
@@ -0,0 +1 @@
ts":[{"start_timestamp":"2022-09-05T13:25:38.188Z","line":4,"name":"Testing Address CRI TxMA integration","description":"","id":"whenever-user-creates-an-event-on-address-cri-ui--txma-s3-bucket-should-how-a-log-of-this-event-within-a-minute.;testing-address-cri-txma-integration","type":"scenario","keyword":"Scenario","steps":[{"result":{"duration":7813356000,"status":"passed"},"line":5,"name":"the user is on Address CRI","match":{"location":"uk.gov.di.txma.audit.bdd.AddressCRI_TxmaStepDefinitions.the_user_is_on_Address_CRI()"},"keyword":"Given "},{"result":{"duration":1353735000,"status":"passed"},"line":6,"name":"the user enters their postcode and click `Find address`","match":{"location":"uk.gov.di.txma.audit.bdd.AddressCRI_TxmaStepDefinitions.the_user_enters_their_postcode_and_click_Find_address()"},"keyword":"When "},{"result":{"duration":809883000,"status":"passed"},"line":7,"name":"the user chooses their address from dropdown and click `Choose address`","match":{"location":"uk.gov.di.txma.audit.bdd.AddressCRI_TxmaStepDefinitions.the_user_chooses_their_address_from_dropdown_and_click_Choose_address()"},"keyword":"When "},{"result":{"duration":814869000,"status":"passed"},"line":8,"name":"the user enters the date they moved into their current address","match":{"location":"uk.gov.di.txma.audit.bdd.AddressCRI_TxmaStepDefinitions.the_user_enters_the_date_they_moved_into_their_current_address()"},"keyword":"When "},{"result":{"duration":1726343000,"status":"passed"},"line":9,"name":"the user clicks `I confirm my details are correct`","match":{"location":"uk.gov.di.txma.audit.bdd.AddressCRI_TxmaStepDefinitions.the_user_clicks_I_confirm_my_details_are_correct()"},"keyword":"When "},{"result":{"duration":313435000,"status":"passed"},"line":10,"name":"Response from Address CRI Integration displays the user's address in JSON","match":{"location":"uk.gov.di.txma.audit.bdd.AddressCRI_TxmaStepDefinitions.response_from_Address_CRI_Integration_displays_the_user_s_address_in_JSON()"},"keyword":"Then "},{"result":{"duration":32146000,"status":"passed"},"line":11,"name":"the user clicks on summaryTest and reads the sub value from JSON","match":{"location":"uk.gov.di.txma.audit.bdd.AddressCRI_TxmaStepDefinitions.the_User_Clicks_On_Summary_Test_And_Reads_The_Sub_Value_From_JSON()"},"keyword":"When "},{"result":{"duration":36205689000,"status":"passed"},"line":12,"name":"the audit S3 should have a new event with the postcode provided","match":{"location":"uk.gov.di.txma.audit.bdd.AddressCRI_TxmaStepDefinitions.the_Audit_S3_Should_Have_A_New_Event_With_The_Postcode_Provided()"},"keyword":"Then "}],"tags":[{"name":"@staging"}]}],"name":"Whenever user creates an event on Address CRI UI, TxMA S3 bucket should how a log of this event within a minute.","description":"","id":"whenever-user-creates-an-event-on-address-cri-ui--txma-s3-bucket-should-how-a-log-of-this-event-within-a-minute.","keyword":"Feature","uri":"classpath:features/AddressCRI_Txma.feature","tags":[]}]
```

Comment:
Both these null files should be removed

Reply:
These files comeback automatically after every run even if we remove it. Not sure if I am missing something.